src/media/js/main.js

```diff
@@ -332,5 +336,118 @@ function(_) {
         throw new Error('Cancel navigation; deprecated client');
     });
 
+    var desktopPromo = document.querySelector('.desktop-promo-items');
```

Comment:
I'm going to make this show up only if the device_type is desktop. I'll move it.